admin/services/restart.service.js

```diff
@@ -110,4 +111,47 @@ restartService.canRestart = async pupilId => {
   return hasUsedRestart && hasRestartAttemptRemaining && hasCheckAttemptRemaining
 }
 
+/**
+ * Get pupils who have been submitted for a restart
+ * @param schoolId
+ * @returns {Array}
+ */
+
+restartService.getSubmittedRestarts = async schoolId => {
+  let pupils = await pupilDataService.getSortedPupils(schoolId, 'lastName', 'asc')
+  if (!pupils || pupils.length === 0) return []
+  let restarts = []
+  await Promise.all(pupils.map(async p => {
+    const r = await pupilRestartDataService.findOne({ pupilId: p._id, isDeleted: false })
```

Comment:
It's worth adding a comment that the findOne is inside the loop because of a bug in the Cosmos MongoDB API.  We will want to fix this later. So a TODO would be correct...

Reply:
updated